lambdas/service/app.py

```diff
@@ -183,36 +183,48 @@ def openapi():
 
 health_spec = {
     'responses': {
-        '200': {
-            'description': '200 response',
+        f'{200 if up else 503}': {
+            'description': f'Service component is {"" if up else "un"}reachable.',
             'content': {
                 'application/json': {
-                    'schema': {'$ref': '#/components/schemas/Empty'}
+                    'schema': schema.object(
+                        additional_properties=schema.object(
+                            additional_properties=True,
+                            up=schema.enum(up)
+                        ),
+                        up=schema.enum(up)
+                    )
                 }
             }
-        }
+        } for up in [True, False]
     },
     'tags': ['Auxiliary']
 }
 
 
 @app.route('/health', methods=['GET'], cors=True, method_spec={
     **health_spec,
-    'summary': 'List health information for webservice'
+    'description': 'List health information for Azul service',
+    'summary': 'Check Azul health'
 })
 @app.route('/health/{keys}', methods=['GET'], cors=True, path_spec={
     'parameters': [
         {
             'name': 'keys',
             'in': 'path',
             'required': True,
-            'schema': {'type': 'string'}
+            'description': 'Available component(s) to query health check status: '
```

Comment:
Use `all_keys` and `internal_keys` attributes to derive this line.

Reply:
I also think we shouldn't introduce "component" as a synonym for the _keys_ concept. If we do, we need to strongly associate the two in the description so users understand that they are the same thing. But again, I'd rather not introduce another alias, especially since it does not completely fit. For example, `fast` isn't a component. Try to find a description that ties the _keys_ parameter to the fact that it selects certain entries for the response body. Writing concise and accurate documentation is difficult, so you'll need to think about this very hard.